css-viewport/Overview.bs

```diff
@@ -356,6 +356,154 @@ only the value previously set to it.
 }
 </pre>
 
+<h2 id='zoom-property'>
+The 'zoom' property</h2>
+
+An element becomes zoomed when the 'zoom' property has a positive computed value
+different than 1. 'zoom' affects computed property values and inherited
+property values. The computed value of 'zoom' is applied as a scalar to
+the used value &lt;length&gt;s CSS properties for the element and
+its descendants (including inherited properties such as 'line-height' or
+'font-size'), plus intrinsic sizing of
+all replaced elements and nested frames, resulting in a magnification or
+minification effect.
+
+Nested values of 'zoom' multiply, resulting in additional scaling
+of &lt;length&gt; values.
+
+The 'zoom' property has no effect on &lt;length&gt; property values with
+computed values that are ''auto'' or &lt;percent&gt;.
+
+<div class="note">Unlike 'transform', scaling the 'zoom' property affects
+layout.</div>
+
+<p class="note"> The computed value of 'font-size' is never &lt;percent&gt;
+thus 'zoom' always applies.</p>
+
+<p class="note">'zoom' does not affect or prevent 'transform' scaling.</p>
+
+<pre class="propdef">
+Name: zoom
+Value: &lt;number&gt;
+Initial: 1
+Applies to: all &lt;length&gt; property values of block-level elements
+Inherited: no
+Percentages: Refer to all &lt;length&gt; property values
+Media: visual
+Computed value: as specified
+</pre>
+
+The values of this property have the following meanings:
+
+<dl dfn-for="wrap-flow" dfn-type=value>
+  <dt><dfn lt="'zoom'!!'number'">number</dfn>
```

Comment:
Leave out the lt and put `<<number>>`. But fix the dfn-for to say "zoom"!

Reply:
I'm curious what spec you got this pattern from, @chrishtr, because it's an *extremely* old pattern that triggered some special behavior in the pre-Bikeshed processor (which Bikeshed now does by default). If there's still some of this hanging around the repo we need to fix it.

But yeah, fix your `dfn-for` attribute too. ^_^

Reply:
Done. As for where I got this old syntax from, it's because I started out with the text from Rossen's unofficial spec draft on his personal repo, which was from many years ago.